app/controllers/album.js

```diff
@@ -0,0 +1,30 @@
+const albumsService = require('../services/albums'),
+  logger = require('../logger'),
+  message = require('../constants');
+
+exports.getAlbumsInventory = (req, res, next) => {
+  logger.info(`${message.PREVIOUS_MESSAGE} to list of albums`);
+  albumsService
+    .getAlbums()
+    .then(response => {
+      logger.info(message.MESSAGE_OK);
+      res.status(200).send(response);
```

Comment:
Here is not necessary to add a `return` because is the last line of the function, but always add it to make the code clearer and to avoid future errors. Another programmer can make more code in this function.